graphstorm-processing/graphstorm_processing/distributed_executor.py

```diff
@@ -119,6 +119,8 @@ class ExecutorConfig:
         The name of the graph being processed. If not provided we use part of the input_prefix.
     do_repartition: bool
         Whether to apply repartitioning to the graph on the Spark leader.
+    num_s3_threads: int
+        Number of threads to write the files to S3.
```

Comment:
Let's say for S3 interactions here. get_row counts e.g. only reads from S3, doesn't write.